server/controllers/notification.js

```diff
@@ -0,0 +1,50 @@
+const Notification = require("../models/Notification");
+const asyncHandler = require("express-async-handler");
+// createNotification, readNotification, getAll, getUnread
+exports.createNotification = asyncHandler(async (req, res) => {
+    const notification = JSON.parse(req.body)
+    try{
+        notification.save()
+        .then(res.status(200));
+    } catch (error) {
+        res.json(error);
+    }
+});
+exports.readNotification = asyncHandler(async (req, res) => {
+    const { id } = req.params;
+    try {
+        Notification
+        .updateAll({ receivedBy: id }, { $set: {  read: true } })
+        .then(res.status(200));
+    } catch (error) {
+        res.json(error);
+    }
+});
+exports.getAll = asyncHandler(async (req, res) => {
+    const { id } = req.params;
```

Comment:
We can use the request object to obtain the logged in user's id - `req.user.id`

Ditto below